### DISCUSSION TITLE
How do I buy the SBC?

### DISCUSSION THREAD

**Discussion by eiger3970** [AvaotaSBC/.github#5] [General]
Hello, I've looked around, however I can't find where to buy the SBC hardware?

**Comment by SamulKyull** [comment]
International sales have not yet begun. If you are within mainland China, you can purchase from the following link: https://detail.tmall.com/item.htm?&id=796901435546

**Comment by BruceBushby** [comment]
Any updates on how to buy globally .... in my case the UK ?  What is the cost estimate to chop up the working design for modules in "CM" and "DIMM" format ? .... what are the challenges squeezing two core on one dimm ?  If I can get hold of a board I can at least help following the docs and building/running images etc. Awesome project, cheers for all the hard work!!!

**Reply by SamulKyull** [reply]
- https://pine64.com/product/yuzuki-avaota-a1-single-board-computer-4gb-32gb/
- https://pine64.com/product/yuzuki-avaota-a1-single-board-computer-2gb-16gb/

**Comment by thanhtantran** [comment]
I have board 4GB/32GB, anyone need, plaese email me
